manifests/m/mazurwiktor/AlbionOnlineStats/0.11.12/mazurwiktor.AlbionOnlineStats.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: mazurwiktor.AlbionOnlineStats
+PackageVersion: 0.11.12
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/mazurwiktor/albion-online-stats/releases/download/0.11.12/Albion_Online_Stats_0.11.12.exe
+  InstallerSha256: E78CCA61574A63478F04F2E6A5491BE2CFE5650B7BDE2EA8EE440EFF2F680828

```

```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/mazurwiktor/albion-online-stats/releases/download/0.11.12/Albion_Online_Stats_0.11.12.exe
  InstallerSha256: E78CCA61574A63478F04F2E6A5491BE2CFE5650B7BDE2EA8EE440EFF2F680828
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/mazurwiktor/albion-online-stats/releases/download/0.11.12/Albion_Online_Stats_0.11.12.exe
  InstallerSha256: E78CCA61574A63478F04F2E6A5491BE2CFE5650B7BDE2EA8EE440EFF2F680828
  InstallerSwitches:
    Custom: /ALLUSERS
```
Once you're at it you can add this